content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,22 @@
+# Getting Started with Entity Framework Core
+
+## Who is this tutorial meant for?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
+
+If you know what you're looking for, feel free to browse by topic on the side. If not, welcome! Let's get started together!
+
+## Prerequisites for this tutorial
+
+You should understand the following concepts before starting this tutorial:�
+
+* C# (see the [interactive C# tutorial](https://www.microsoft.com/net/tutorials/csharp/getting-started))
```

Comment:
Here is an excerpt from the ASP.NET tutorial:

"This tutorial is aimed at people who are learning ASP.NET Core for the first time. Familiarity with C# is assumed. If you're new to C#, you may wish to go through the Getting Started with C# tutorial before beginning this one."